From 771d5a887a0b5975128c4a699e51df1ab0069141 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Fri, 12 Aug 2022 17:03:57 +0800
Subject: [PATCH 22/46] Revert "drm/msm/hdmi: check return value after calling
 platform_get_resource_byname()"

This reverts commit 2b3ed7547b1a052209da6c4ab886ffe0eed88c42.
---
 drivers/gpu/drm/msm/hdmi/hdmi.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/msm/hdmi/hdmi.c b/drivers/gpu/drm/msm/hdmi/hdmi.c
index a0fd62b6ec99..1a7e77373407 100644
--- a/drivers/gpu/drm/msm/hdmi/hdmi.c
+++ b/drivers/gpu/drm/msm/hdmi/hdmi.c
@@ -142,10 +142,6 @@ static struct hdmi *msm_hdmi_init(struct platform_device *pdev)
 	/* HDCP needs physical address of hdmi register */
 	res = platform_get_resource_byname(pdev, IORESOURCE_MEM,
 		config->mmio_name);
-	if (!res) {
-		ret = -EINVAL;
-		goto fail;
-	}
 	hdmi->mmio_phy_addr = res->start;
 
 	hdmi->qfprom_mmio = msm_ioremap(pdev,
-- 
2.27.0

